docs/developers/documentation/index.md

```diff
@@ -175,41 +202,47 @@ also be caught when previewing the built documentation. If your documentation
 change does not include code, you only need the napari docs dependencies
 installed.
 
-There are two ways you can preview the documentation website, by building
-locally or downloading the GitHub Actions built documentation when you submit
+There are two ways you can preview the documentation website: by building
+locally, or downloading the GitHub Actions built documentation when you submit
 your pull request.
 
-To build the documentation locally, run `make docs` from the root of
-the `napari` repository (assuming you've installed the [docs prerequisites](#prerequisites)).
+To build the documentation locally, run `make docs` from the root of your local
+clone of the `napari/docs` repository (assuming you've installed the [docs prerequisites](#prerequisites)).
 
 ```bash
 make docs
 ```
 
-The rendered HTML will be placed in `napari/docs/_build`. Find `index.html` in this folder and drag it
+```{note}
+The `make docs` command above assumes your have a local clone of the
+[`napari/napari`](https://github.com/napari/napari) repo at the same level as
+the `napari/docs` clone.
+```
```

Comment:
Not the best solution, but the Makefile needs to be updated in a follow-up, and this can be corrected accordingly.